.github/workflows/deploy-docs.yaml

```diff
@@ -28,7 +28,7 @@ jobs:
         run: pipx install poetry
 
       - name: Install dependencies.
-        run: poetry install -E docs
+        run: poetry install --with docs
```

Comment:
this was the command that was causing the problem -- needed to be `--with` instead of `-E`, which refers to `extras`.